assets/helpers/page/page.js

```diff
@@ -49,6 +57,16 @@ function analyticsInitialisation(participations: Participations): void {
   ga.setDimension('experience', abTest.getVariantsAsString(participations));
   ga.trackPageview();
 
+  const dimensions:Dimensions = {
+    campaignCodeBusinessUnit: getQueryParameter('CMP_BUNIT') || undefined,
+    campaignCodeTeam: getQueryParameter('CMP_TU') || undefined,
+    experience: abTest.getVariantsAsString(participations),
+  };
+
+  if (!(doNotTrack() === '1' || doNotTrack() === 'yes')) {
```

Comment:
`navigator.doNotTrack` is an [experimental feature](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/doNotTrack). Nevertheless, it should work for every browser, if the browser doesn't support it this function will return undefined and we will `init` the `googleTagManager` module.